benchmarks/build.gradle

```diff
@@ -1,4 +1,12 @@
-apply plugin: 'com.google.protobuf'
+buildscript {
+    repositories {
+        gradlePluginPortal()
```

Comment:
I haven't seen this before because plugin portal and jcenter should be there by default. Do you need it?

Reply:
It seems like the build fails if I comment out `repositories { ... }`:

```
Cannot resolve external dependency me.champeau.gradle:jmh-gradle-plugin:0.4.5 because no repositories are defined.
     Required by:
         project :benchmarks
```